rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py

```diff
@@ -5,7 +5,15 @@
 def cleanup_tokens(
     token_ids_string: List[Tuple[int, Text]], delimiter: Text
 ) -> Tuple[List[int], List[Text]]:
-    """Utility method to apply specific delimiter based cleanup on list of tokens"""
+    """Utility method to apply delimiter based cleanup on list of tokens.
+
+    Args:
+        token_ids_string: List of tuples with each tuple containing (token id, token string).
```

Comment:
Just for my understanding: token id = position of token in that message? Or token id = id in vocab?

Reply:
id in vocab

Reply:
minor: should we add it somewhere? it is mentioned  couple of times, maybe when we create those ids?

Reply:
added 👍